_data/nav.yml

```diff
@@ -478,11 +478,13 @@
     - url: "#template_mdate_classes"
     - url: "#template_mdate_format"
     - url: "#template_replace_values"
+    - url: "#template_preview_replace_values"
     - url: "#template_selected_content_classes"
     - url: "#Template Plugin Examples"
     - url: "#Making Templates"
     - url: "#Making Snippets"
     - url: "#Reference Date/Time formats"
+    - url: "#Example Configuration"
```

Comment:
unfortunately due to constraints of the current system, we typically only have the options in the Nav :( please remove